docs/mesos/UsageGuideUnifiedCNI.md

```diff
@@ -30,44 +30,102 @@ EOF
+Mesos-Agent loads these network configurations into memory at launch, so you will need to restart the Agent process for your new configuration to take effect.
 
 ## 2. Launch a Calico-Networked Task
-#### Using Mesos-Execute
-With our network configured, we can use `mesos-execute` to launch a task on the CNI network you just configured by setting `--networks` to the name of the network you just configured:
+#### a.) Using Mesos-Execute
+To quickly test that Calico's Mesos integration is working, use `mesos-execute` to launch a task on the created CNI network by setting `--networks`:
 ```
 mesos-execute --containerizer=mesos \
-              --name=cni \
+              --name=calico-cni-base-test \
               --master=172.17.0.4:5050 \
               --networks=calico-net-1 \
               --command="ifconfig"
 ```
+The task will have been successfully networked by Calico if its return code is `0`. Additionally, txhe task's `stdout` log should show an IP address from the default Calico pool: `192.168.0.0/16`
 
-#### Using Marathon
-Create `app.json` with `ipAddress.networkName` set to the CNI network:
+#### b.) Using Marathon
+In Marathon v1.2.0+, set the task's `ipAddress.networkName` to the name of the CNI network:
 ```
+curl -X POST -H "Content-Type: application/json" http://localhost:8080/v2/apps -d '
 {
-    "id": "frontend",
+    "id": "calico-cni-marathon-test",
     "cmd": "ifconfig && sleep 1000",
     "cpus": 0.1,
     "mem": 64.0,
     "ipAddress": {
         "networkName": "calico-net-1",
         "labels": {
+          "app": "test",
+          "group": "development"
+        }
+    }
+}'
+```
+Again, this task should show a Calico IP in the Marathon UI.
+#### c.) Using Marathon - Unified Containerizer
+
+With the release of Mesos-1.0, the Unified Containerizer can now launch Docker Images using the Mesos Containerizer. Calico can perform networking for these tasks.
+
+Requires:
+- Mesos v1.0.0+
+> To make use of this feature, ensure you are using Mesos 1.0.0+ with agents [configured to use the Unified Containerizer](http://mesos.apache.org/documentation/latest/container-image/). Also ensure you are using Marathon v1.3.0+.
+
+Our `app.json` looks similar to the one above, but with a few extra settings (and some more practical use):
+```
+curl -X POST -H "Content-Type: application/json" http://localhost:8080/v2/apps -d '
+{
+    "id": "frontend",
+    "container": {
+      "type": "MESOS",
+      "docker": {
+        "image": "nginx"
+      }
+    },
+    "cpus": 0.1,
+    "mem": 64.0,
+    "ipAddress": {
+        "networkName": "calico-net-1",
+        "labels": {
           "app": "frontend",
           "group": "production"
         }
     }
-}
+}'
+```
+
+The application will show its Calico IP when viewed using the Marathon UI, however it will be unreachable until Calico policy is configured to allow traffic.
+
+## 3. Configuring Policy
```

Comment:
I refrained from being too descript here because I don't want this doc to be the place this stuff is "announced."